addons/web/static/tests/views/graph_view_tests.js

```diff
@@ -1219,6 +1219,52 @@ QUnit.module("Views", (hooks) => {
 
         assert.strictEqual(graph.model.metaData.mode, "line", "should be in line chart mode.");
         assert.strictEqual(graph.model.metaData.cumulated, true, "should be in cumulative");
+        assert.strictEqual(
+            graph.model.metaData.cumulated_start,
+            false,
+            "should have cumulated start opted-out"
+        );
+    });
+
+    QUnit.test("Cumulative prop and cumulated start", async function (assert) {
+        const graph = await makeView({
+            serverData,
+            type: "graph",
+            resModel: "foo",
+            arch: `
+                <graph type="line" stacked="0" cumulated="1" cumulated_start="1">
+                    <field name="date"/>
+                    <field name="product_id"/>
+                </graph>
+            `,
+            comparison: {
```

Comment:
it would be better to use a search arch with a simple filter and set it as search_default.